string name
string family
int64 macaddress
